src/contents/articles/hello-world/index.md

```diff
@@ -27,4 +25,4 @@ We want to grow as software developers but we also want to grow as a people. As
 
```

Comment:
I would remove the section - **The namespace way of life** because it mentions the pillars and the mantra without exploring them in enough detail to justify their inclusion. I don't think either need to be mentioned or explained in an introductory post.